### ISSUE TITLE
segmentation fault: 11 on conda in terminal on Mac OS X (Navigator is not launching)

### ISSUE THREAD

**Issue by LiuuXiaoMeng** [ContinuumIO/anaconda-issues#10816]
I tried to include dlib into my anaconda environment through the following command:
`conda install -c menpo dlib`
After I installed dlib, I couldn't launch my Anaconda Navigator. It just quit suddenly. 
The following screen shot refers to "conda list --revisions"results:

![屏幕快照 2019-04-10 下午3 18 52](https://user-images.githubusercontent.com/34299182/55859605-fc780900-5ba4-11e9-93cd-4c61be0c2080.png)
These operations were automatically done during I installed dlib. I think it must be the red underlined operation cause this problem. But I don't know how to fix it.
I search the Internet and tired some commands. I have already update my anaconda, but it still cannot launch. And I tried this command in terminal:
`anaconda-navigator --reset`
However the result I got was:
`Segmentation fault: 11`
I really don't know where went wrong, and don't know how to fix it.  Could anyone tell me how to fix it?
Thank you sooo much!!

**Comment by LiuuXiaoMeng** [comment]
Is there anyone can help me with this problem? I'm really dying for a  solution...(TAT)

**Comment by oskwys** [comment]
Did you manage to solve it? I have the same problem...

**Comment by LiuuXiaoMeng** [comment]
> Did you manage to solve it? I have the same problem...

Nope, but I change the way I start Anaconda. You can use terminal in PyCharm to navigate Anaconda.

**Comment by alanbchristie** [comment]
Same here - for some reason my conda installation is now corrupt and all I get when trying to adjust or create environments or update conda itself is `Segmentation fault: 11`: -

```
$ conda update conda
Collecting package metadata (current_repodata.json): done
Solving environment: done
Segmentation fault: 11
```
